linkerd/service-profiles/src/http/mod.rs

```diff
@@ -26,7 +26,7 @@ pub enum RequestMatch {
     All(Vec<RequestMatch>),
     Any(Vec<RequestMatch>),
     Not(Box<RequestMatch>),
-    Path(Regex),
+    Path(Box<Regex>),
```

Comment:
I suspect the size of `Regex` changed and now clippy is unhappy about the size of the enum variant

Reply:
Yeah, it probably used to box internally, I guess?